docs/snippets/azure/AppHost/Program.ConfigureWebPubSubInfra.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Azure.Provisioning.WebPubSub;
+
+internal static partial class Program
+{
+    public static void ConfigureWebPubSubInfra(IDistributedApplicationBuilder builder)
+    {
+        // <configure>
+        builder.AddAzureWebPubSub("web-pubsub")
+            .ConfigureInfrastructure(infra =>
+            {
+                var webPubSubService = infra.GetProvisionableResources()
+                                            .OfType<WebPubSubService>()
+                                            .Single();
+
+                webPubSubService.Sku.Name = "Standard_S5";
```

Comment:
```suggestion
                webPubSubService.Sku.Name = "Standard_S1";
```

![image](https://github.com/user-attachments/assets/72a2207f-7752-4545-b3ee-75ce67780325)